scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala

```diff
@@ -40,7 +41,139 @@ import scala.concurrent.Future
  */
 package object bigtable {
 
-  private val DEFAULT_SLEEP_DURATION = Duration.standardMinutes(20);
+  // =======================================================================
+  // Protobuf helpers
+  // =======================================================================
+
+  /** Helper methods for [[Mutation]]. */
+  object Mutations {
+
+    def newSetCell(family: String, column: ByteString, value: ByteString): Mutation =
```

Comment:
Also without specifying a timestamp it will use `0` (which becomes that date in 1970). It probably makes more sense to help out users by using the current timestamp by default. Its what the hbase api did and will help users switch over.